lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +194,13 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    period = random.randrange(5)
+    time.sleep(period)
+
+    response = requests.get(url=url, timeout=config.get_timeout(),
```

Comment:
Make sure you use all the request configurations given you by `scrapper_config.json`